app/views/team_members/_table.html.erb

```diff
@@ -3,6 +3,21 @@
     <% team_members.each do |member| %>
       <tr class="govuk-table__row">
         <td class="govuk-table__cell "><%= member.email %></td>
+        <td class="govuk-table__cell ">
+	  <% if !member.invitation_sent_at.nil? && !member.invitation_accepted? %>
```

Comment:
I wonder if this should be a method on the `user` model instead, feels like it'd be more expressive there.